spec/support/features/matchers/heading_matcher.rb

```diff
@@ -0,0 +1,24 @@
+RSpec::Matchers.define :have_heading do |tag, text|
+  match do |page|
+    # Check for the tag with the correct text
+    has_correct_text = page.has_selector?(tag, text: text)
+
+    # If it's an h1, also check that there's only one
```

Comment:
Good thought for accounting for this. Ideally a page shouldn't have more than one `h1` heading.

Reply:
Have more then 1 will fail accessibility :see_no_evil: